forms/core/CampaignMonitor.php

```diff
@@ -37,29 +37,41 @@ public function get_last_error(){
      */
     public function refresh_token($auth = array())
     {
-	   return $this->api->refreshToken();
+        $refreshToken = empty($auth) ? '' : $auth['refresh_token'];

```

Comment:
Just confirming that this code change is required (ie: we need to use the $auth input parameter now)

Reply:
probably not aye, I'll just straight up use the `$auth['refresh_token']`